### PR TITLE
adds a pr template

### PR DESCRIPTION
**Pull request by Da-Colon**
## Description

A straight forward PR template.

## Notes

- Open to discussion and changes
- Let's get everyone's approval to move forward.

## Issue (if applicable)
n/a

## Testing

- Will test once merged in develop and next PR is open.
- should have useful sections for reviewer
- should have useful descriptions for author

## Screenshots (if applicable)
n/a

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Description
+
+<!-- Please describe your changes -->
+
+## Notes
+
+<!-- Is there any additional information a reviewer should know?  -->
+
+
```
Comment (adamgall) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: Remove extra newline here lol

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## Issue (if applicable)
+
+<!-- If applicable, please link to the github issue and put `closes #XXXX` in your comment to auto-close the issue that your PR fixes. -->
```
Comment (adamgall) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: Suggest to simplify these instructions

```
<!-- If applicable, link to the relevant issue(s) with `closes #XXX` to auto-close it when this PR is merged -->
```

**Review by adamgall — APPROVED** [review]
Slight changes, but this is great!

**Review by adamgall — APPROVED** [review]

**Review by adamgall — APPROVED** [review]
A thing that I like doing, and I like when you (@Da-Colon) do, is when the PR author goes through their PR changeset on GitHub and leaves comments throughout their code, describing what's going on, and why they decided to build/structure code the way they did.

It might be cool to encourage PR authors to do that, in this template.

**Review by tbwebb22 — APPROVED** [review]
LGTM

**Review by christopherdancy — APPROVED** [review]
LFG

**Comment by Da-Colon** [comment]
LFG 🚀

**Pull request merged**